Samples/src/FSharpWithQSharp/README.md

```diff
@@ -0,0 +1,44 @@
+# Using Q# with F#
+
+This sample shows how to use Q# code with a classical host program written in F#.
+
+*It is also possible to use Q# with VB.NET; for an example, see [this blog post](https://github.com/tcNickolas/MiscQSharp/tree/master/Quantum_VBNet).*
+
+Q# code is compiled to C# for the purposes of local simulation, so using Q# with C# is very straightforward, as described in the documentation. Using Q# with F# is a little less obvious, since you can't mix Q# and F# files in the same project. You have to create a Q# library (which will yield a .csproj project with only Q# files in it) and then reference it from a purely F# application.
+
+The steps are as follows:
+
+1. Create a Q# library `QuantumCode` and write your quantum code in it.
+2. Create an F# application (in this case a console app targeting .NET Core) `FsharpDriver`.
+3. Add a reference to the Q# library to the F# application.
+
+   You can use [Reference Manager](https://docs.microsoft.com/visualstudio/ide/how-to-add-or-remove-references-by-using-the-reference-manager) in Visual Studio to do that, or you can add the reference from the command line:
+
+    ```PowerShell
+    PS>  dotnet add .\FsharpDriver\FsharpDriver.vbproj reference .\QuantumCode\QuantumCode.csproj
+    ```
+   
+   This will transitively include   `Microsoft.Quantum.Development.Kit` NuGet package to the F# application.
```

Comment:
```suggestion
   This will transitively include the [`Microsoft.Quantum.Development.Kit` NuGet package](https://www.nuget.org/packages/Microsoft.Quantum.Development.Kit) to the F# application.
```